apps/src/music/views/MusicView.jsx

```diff
@@ -175,9 +175,9 @@ class UnconnectedMusicView extends React.Component {
 
   updateTimer = () => {
     if (this.props.isPlaying) {
-      this.setState({
-        currentPlayheadPosition: this.player.getCurrentPlayheadPosition()
-      });
+      this.props.setCurrentPlayheadPosition(
+        this.player.getCurrentPlayheadPosition()
+      );
```

Comment:
Would it make sense for the player to do this directly with its own timer?

Reply:
Yeah good question, I wasn't sure where this dispatch should happen. I think keeping this in the view still does make sense to me, so that the view can decide how often it wants to update the playhead UI state, which I feel like the player doesn't need to worry about? I do think as a follow-up though, that I will pull the timer logic into a separate sub-component just to start pulling out logic from MusicView.

Reply:
> Yeah good question, I wasn't sure where this dispatch should happen. I think keeping this in the view still does make sense to me, so that the view can decide how often it wants to update the playhead UI state, which I feel like the player doesn't need to worry about?

That makes sense!